src/server/rest/v1/service/UserService.ts

```diff
@@ -1106,6 +1128,15 @@ export default class UserService {
     }
     // Delete the Tag
     await UserStorage.deleteTag(req.user.tenantID, tag.userID, tag);
+    const user = await UserStorage.getUser(req.user.tenantID, tag.userID,{ withTag: true });
```

Comment:
I would rather get the tags with the userID + only one record (more efficient if you have hundreds badge), you check if the count is === 1 and set the only first record in the result as default badge ;-)

Reply:
Done